#!/usr/bin/env python
PACKAGE = "jaco2_driver"
import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()



gen.add("trajectory_p_gain_joint_0", double_t, 0, "follow trajactory p gain for joint 0", 11.29063216, -100, 100)
gen.add("trajectory_p_gain_joint_1", double_t, 0, "follow trajactory p gain for joint 1", 11.29063216, -100, 100)
gen.add("trajectory_p_gain_joint_2", double_t, 0, "follow trajactory p gain for joint 2", 11.29063216, -100, 100)
gen.add("trajectory_p_gain_joint_3", double_t, 0, "follow trajactory p gain for joint 3", 11.29063216, -100, 100)
gen.add("trajectory_p_gain_joint_4", double_t, 0, "follow trajactory p gain for joint 4", 11.29063216, -100, 100)
gen.add("trajectory_p_gain_joint_5", double_t, 0, "follow trajactory p gain for joint 5", 11.29063216, -100, 100)


exit(gen.generate(PACKAGE, "jaco2_driver", "jaco2_driver_configure"))
